securedrop/dictionaries/nouns.txt

```diff
@@ -15548,15 +15277,12 @@ spelling
 spender
 spending
 spendthrift
-sperm
-spermicide
 sphagnum
 sphere
 sphericity
 spheroid
 sphincter
```

Comment:
Let's drop this one, lest we ["Wayne's World"](http://youtu.be/--BslU44vWU) some unsuspecting journalists...